### DISCUSSION TITLE
horus.exe

### DISCUSSION THREAD

**Discussion by edugardo1** [community/community#33213] [Other Feature Feedback, Questions, & Ideas]
desaparecio el archivo del software horus.exe. que paso? si el software era libre? tengo un scaner siclop he3d y no tengo como hacerlo funcionar.

**Comment by jfullstackdev** [comment]
you mean this ? 

https://github.com/LibreScanner/horus

fortunately it's open source

**Comment by edugardo1** [comment]
thanks for answering.
if I refer to that program, but I can not download the interface.
I need the horus.exe software. that is, the executable to be able to manage
the system in windows, including the camera, the lasers and the motor.
When I try to download it, this page appears and I don't know how to
continue.
[image: image.png]
and when trying to install horus....
[image: image.png]
gracias

**Comment by jfullstackdev** [comment]
unfortunately, the last commits were last 2016, 

maybe what you can do is to check the contributors one by one, and try contacting them, but again, there is still no assurance

**Comment by edugardo1** [comment]
Ok, thanks.
